.github/workflows/fetch-merged-prs.yml

```diff
@@ -0,0 +1,41 @@
+name: Fetch Merged PRs After Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release after which to fetch merged PRs'
+        required: true
+      branch:
+        description: 'Branch to fetch merged PRs from'
+        required: true
+        default: 'main'
+
+jobs:
+  fetch-merged-prs:
+    runs-on: ubuntu-latest
+
```

Comment:
Remove empty line

Reply:
Why? It will make it less readable. Same for all the rest of these comments.

Reply:
Our code style keeps things as compact as possible.

Reply:
We'll come up with a shared code style after this pr.